src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,185 @@
+  It adds background effects like blurring and transparency to
+  the artefacts behind an element.
+tags:
+  - post
+  - css
+  - backdrop-filter
+---
+Translucence, blurring, and other effects are useful ways of creating depth while keeping the context of the background content. They support a host of use cases such as frosted glass, video overlays, translucent navigation headers, inappropriate image censoring, image loading, and so on. You may recognize these effects from two popular operating systems: [Windows 10](https://i.kinja-img.com/gawker-media/image/upload/s--9RLXARU4--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/trgz8yivyyqrpcnwscu5.png) and [iOS](https://static.businessinsider.com/image/51fd2822eab8eae16e00000b-750.jpg).
+
+<figure class="w-figure w-figure--center">
+  <img src="weather_app.jpg" alt="An example of a frosted glass effect.">
+  <figcaption class="w-figcaption">
+    An example of a frosted glass effect. <a href="https://dribbble.com/shots/733714-Weather-App?list=tags&tag=android" target="_blank">Source</a>.
+  </figcaption>
+</figure>
+
+Historically, these techniques were difficult to implement on the web, requiring less than perfect [hacks or workarounds](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter). In recent years both [Safari](https://webkit.org/blog/3632/introducing-backdrop-filters/) and Edge have provided these capabilities through the `background-filter` (and alternatively, the `-webkit-backdrop-filter`) property which dynamically blends foreground and background colors based on filter functions. Now Chrome supports `background-filter`, starting in version 76.
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-kitchen_sink.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    A demonstration of the filter functions for <code>backdrop-filter</code>. Try the example on <a href="https://codepen.io/robinrendle/pen/LmzLEL" target="_blank">CodePen</a>.
+  </figcaption>
+</figure>
+
+## Basics
+
+- The `backdrop-filter` property applies one or more filters to an element, changing the appearance of anything behind the element.
+- The overlaying element must be at least partially transparent.
+- The overlaying element will get a new stacking context.
+
+{% Aside 'caution' %}
+`backdrop-filter` may harm performance. Test it before deploying.
+{% endAside %}
+
+CSS `backdrop-filter` applies one or more filters to the backdrop of an element, leaving the contents unchanged. Some people find its name confusing. Is it changing the background? It does not. Rather, it alters the appearance of whatever is seen through it. It almost goes without saying that if the element has no transparency, that is, if its value is set to `none` there will be no effect on the background.
```

Comment:
I believe this still isn't quite right. Whether or not the value of `backrop-filter` is set to `none` doesn't affect transparency. You need to do that another way--by setting `opacity`, having no background, or having a semi-transparent background.

Reply:
@argyleink The text in question is almost verbatim from your draft. Can you please comment and help us correct?

Reply:
i see, consider this? 
"It almost goes without saying that if you cannot see through the element at all, there will be no effect from `backdrop-filter`."

haha, i dont feel like i was a good tie-breaker here. but this part of the copy is what i agree isn't correct yet "that is, if its value is set to `none` there will be no effect on the background"

Reply:
Took another crack at it.